modules/SofaSparseSolver/src/SofaSparseSolver/PrecomputedLinearSolver.h

```diff
@@ -134,7 +134,7 @@ private :
 };
 
 #if !defined(SOFA_COMPONENT_LINEARSOLVER_PRECOMPUTEDLINEARSOLVER_CPP)
-extern template class SOFA_SOFASPARSESOLVER_API PrecomputedLinearSolver< CompressedRowSparseMatrix<double> , FullVector<double> >;
+extern template class SOFA_SOFASPARSESOLVER_API PrecomputedLinearSolver< linearalgebra::CompressedRowSparseMatrix<double> , FullVector<double> >;
 #endif
```

Comment:
```suggestion
extern template class SOFA_SOFASPARSESOLVER_API PrecomputedLinearSolver< linearalgebra::CompressedRowSparseMatrix<double> , linearalgebra::FullVector<double> >;
#endif
```
Well spotted, as it could be only detected if somebody else than PrecomputedLinearSolver.cpp is including this header.
But shoud FullVector not be prefixed with linearalgebra namespace as well?

Reply:
Yes you are right. I don't know why but only modifying the first one worked for me. It was certainly not the reason why it didn't compile in the first place. But well, at least I've spotted half of the problem...

Reply:
could be something before this head doing a typedef or something like that maybe.
I am adding the suggestion and will merge afterwards